astropy/modeling/tests/test_separable.py

```diff
@@ -0,0 +1,114 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+Test separability of models.
+
+"""
+
+from .. import models
+from ..models import Mapping
```

Comment:
Nitpick: Group all the local imports together to make it easier for the future. (I am starting to sound like `flake8` 😱 )